BlazorUI/wwwroot/css/open-iconic/README.md

```diff
@@ -0,0 +1,114 @@
+[Open Iconic v1.1.1](http://useiconic.com/open)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Could not find module `remark-lint-no-unneeded-full-reference-image`](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5088867)